File: c/1792/1791.c
int findCenter(int** edges, int edgesSize, int* edgesColSize) {
  int v0 = edges[0][0];
  int v1 = edges[0][1];
  return edges[1][0] == v0 || edges[1][1] == v0 ? v0 : v1;
}

File: ts/1792/1791.ts
// 寻找公共节点
function findCenter(edges: number[][]): number {
  let v0 = edges[0][0], v1 = edges[0][1]
  if (edges[1][0] === v0 || edges[1][1] === v0) return v0
  return v1
};